### PR TITLE
HazelcastListFactoryBean

### PR DESCRIPTION
**Pull request by aaron-brown**
Includes updates to **HazelcastInstanceService** and minor corrections to **HazelcastMapFactoryBeanSpec**.

### CODE REVIEW

File: src/main/groovy/com/budjb/hazelcast/HazelcastInstanceService.groovy [hunk 1]
```diff
@@ -129,4 +129,21 @@ class HazelcastInstanceService implements InitializingBean {
...
+     *
+     * @param name Name of the Hazelcast instance.
+     * @throws IllegalArgumentException when no instance exists for the given name.
+     */
+    void shutdownInstance(String name) {
+        getInstance(name).shutdown()
+    }
+
+    /**
+     * SHutdown all Hazelcast instances on <em>this</em> JVM.
```
Comment (budjb) [inline src/main/groovy/com/budjb/hazelcast/HazelcastInstanceService.groovy]: Shutdown has a capital H.

File: src/main/groovy/com/budjb/hazelcast/beans/HazelcastListFactoryBean.groovy [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+     * Name of the hazelcast instance that should contain the map.
+     */
+    String instanceName
+
+    /**
+     * Name of the hazelcast list.
+     */
+    String listName
+
+    @Override
```
Comment (budjb) [inline src/main/groovy/com/budjb/hazelcast/beans/HazelcastListFactoryBean.groovy]: Can you throw
```
/**
 * {@inheritDoc}
 */
```
on these overridden methods?

File: src/main/groovy/com/budjb/hazelcast/HazelcastInstanceService.groovy (hunk 1 above)
Comment (aaron-brown) [inline src/main/groovy/com/budjb/hazelcast/HazelcastInstanceService.groovy]: Addressed (amended commit)

File: src/main/groovy/com/budjb/hazelcast/beans/HazelcastListFactoryBean.groovy (hunk 2 above)
Comment (aaron-brown) [inline src/main/groovy/com/budjb/hazelcast/beans/HazelcastListFactoryBean.groovy]: Addressed (amended commit)

**Pull request merged**